doc/changelogs/CHANGELOG_V14.md

```diff
@@ -71,6 +72,56 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="14.19.2"></a>
+
+## 2022-05-03, Version 14.19.2 'Fermium' (LTS), @BethGriggs prepared by @juanarbol
+
+### Notable Changes
+
+**doc:**
+
+* New release key for Bryan English
+
+Learn more at: https://github.com/nodejs/node/pull/42102
+Contributed by Bryan English (@bengl)
+
+**npm:**
+
+* Upgrade `npm` to `v16.14.17`.
```

Comment:
```suggestion
* Upgrade `npm` to `v6.14.17`.
```

Reply:
This is unbelievable :joy: I will fix this one; the commit message has the same issue. Should I re-run CI for a release commit amending?